test/acceptance/consensus.go

```diff
@@ -13,9 +12,9 @@ import (
 var _ = Describe("a leader node", func() {
 
 	It("must get validations by all nodes to commit a transaction", func(done Done) {
-		test.WithContext(func(ctx context.Context) {
-			// leader is nodeIndex 0, validator is nodeIndex 1
-			network := harness.NewTestNetwork(ctx, 2)
+		consensusAlgos := []consensus.ConsensusAlgoType{consensus.CONSENSUS_ALGO_TYPE_LEAN_HELIX}
```

Comment:
maybe a bit differently:
either extract the logic of creating the algos slice to a function that takes varargs of type `CONSENSUS_ALGO_TYPE`, so that the call would look like:

```golang
harness.WithNetwork(2, withAlgos(consensus.CONSENSUS_ALGO_TYPE_LEAN_HELIX), func(ctx context.Context, network harness.AcceptanceTestNetwork) {
}
```

or maybe create a network builder, like we had in the alpha:
```golang
harness.ANetwork()
  .WithNodes(2)
  .WithAlgos(consensus.CONSENSUS_ALGO_TYPE_LEAN_HELIX)
  .Run(func( network harness.AcceptanceTestNetwork) {
...
})

also, unless I'm missing something, the `context.Context` argument is being unused

Reply:
Fixed in https://github.com/orbs-network/orbs-network-go/pull/63